apps/CA.pl.in

```diff
@@ -127,13 +127,9 @@ if ($WHAT eq '-newcert' ) {
     # create a pre-certificate
     $RET = run("$REQ -x509 -precert -keyout $NEWKEY -out $NEWCERT $DAYS");
     print "Pre-cert is in $NEWCERT, private key is in $NEWKEY\n" if $RET == 0;
-} elsif ($WHAT eq '-newreq' ) {
+} elsif ($WHAT =~ /\-newreq(\-nodes)?/ ) {
```

Comment:
Formally speaking one wants *equivalent* code, and the replacement is not. One has to add `^` in beginning and `$` at the end to make it *equivalent* to the original. Could you throw these two in, please?